src/components/LoginButton/index.tsx

```diff
@@ -9,7 +9,7 @@ import { makeApiURL } from '../../api';
 import { gettext } from '../../utils';
 import styles from './styles.module.scss';
 
-type PublicProps = {};
+type PublicProps = Record<string, unknown>;
```

Comment:
As mentioned above, I'm not sure I understand why we even need `PublicProps` here at all.